db-schema.sql

```diff
@@ -3,9 +3,17 @@ CREATE TABLE "chat_message" (
     "chat_id" uuid REFERENCES "item" ("id") ON DELETE CASCADE,          -- id of item to which chat is attached
     "creator" uuid REFERENCES "member" ("id") ON DELETE SET NULL,       -- sender member
     "created_at" timestamp NOT NULL DEFAULT (NOW() AT TIME ZONE 'utc'),
+    "updated_at" timestamp,                                             -- NULL by default, only set by trigger on update
     "body" character varying(500)
 );
 
-CREATE INDEX ON "chat_message" ("chat_id");     -- optimize lookup by item id
+CREATE INDEX ON "chat_message" ("chat_id");         -- optimize lookup by item id
 
-CREATE INDEX ON "chat_message" ("created_at");  -- optimize order by datetime
+CREATE INDEX ON "chat_message" ("id", "chat_id");   -- optimize lookup for message id and chat id
```

Comment:
I don't really know much, but is it a duplicate for `chat_id`?

Reply:
@pyphilia I wanted to create a compound index for the delete and update query (uses `WHERE` on `chat_id` and `id`). But now that i think about it, maybe we should only use a `WHERE` on `id` since it is already unique and is the primary key. We would then use 'chat_id' only to  check the item permissions... Maybe to discuss later